server/src/api/modules/groups/group.service.js

```diff
@@ -194,10 +194,13 @@ async function create(name, members) {
 
   // Check if every username in the list is valid
   if (members && members.length > 0) {
-    for (let i = 0; i < members.length; i += 1) {
-      if (!playerService.isValidUsername(members[i].username)) {
-        throw new BadRequestError(`Invalid player username: ${members[i].username}`);
-      }
+    const usernamesCheck = await playerService.isValidUsernames(members);
```

Comment:
Maybe this variable could be renamed to "invalidUsernames"?

**Also: isValidUsernames is not an async function so it does not need the "await" before calling it.**